### PR TITLE
chore(zh-cn): translate for `InputEvent`

### PR DESCRIPTION
**Pull request by fuchunhui**
### Description
1. sync content.
2. fix links.

### CODE REVIEW

**Comment by fuchunhui** [comment]
https://github.com/mdn/content/pull/32759

File: files/zh-cn/web/api/inputevent/index.md [hunk 1]
```diff
@@ -3,29 +3,36 @@ title: InputEvent
 slug: Web/API/InputEvent
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/index.md:3]: Please add the meta for l10n, see the [guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE).
```suggestion
slug: Web/API/InputEvent
l10n:
  sourceCommit: 8ca15ba7933a738cf632a9bbf5cfd4e90d1a97b1
```

File: files/zh-cn/web/api/inputevent/index.md [hunk 2]
```diff
@@ -3,29 +3,36 @@ title: InputEvent
...
 ---
 
-{{APIRef("DOM Events")}}
-
-{{SeeCompatTable}}<
+{{APIRef("UI Events")}}
 
 **`InputEvent`** 接口用来构造和字符输入相关的事件对象。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/index.md:8]:
```suggestion
**`InputEvent`** 接口表示通知用户可编辑内容更改的事件。
```

File: files/zh-cn/web/api/inputevent/index.md [hunk 3]
```diff
@@ -3,29 +3,36 @@ title: InputEvent
...
 
-除继承自 {{domxref("UIEvent")}} 和 {{domxref("Event")}} 接口的属性外，还有以下属性：
+- {{domxref("InputEvent.data")}} {{ReadOnlyInline}}
+  - : 返回当前输入的字符串，如果插入文本（例如，当删除字符时）则该值为空字符串。
+- {{domxref("InputEvent.dataTransfer")}} {{ReadOnlyInline}}
+  - : 返回一个 {{domxref("DataTransfer")}} 对象，该对象包含有关要添加到可编辑内容，或从可编辑内容中删除的富文本或纯文本数据的信息。
+- {{domxref("InputEvent.inputType")}} {{ReadOnlyInline}}
+  - : 返回对可编辑内容所做更改的类型，可能的更改包括插入、删除和格式化文本等。
+- {{domxref("InputEvent.isComposing")}} {{ReadOnlyInline}}
+  - : 返回一个 {{JSxRef("Boolean")}} 值，表明该事件是在触发 {{domxref("Element/compositionstart_event", "compositionstart")}} 事件之后且触发 {{domxref("Element/compositionend_event", "compositionend")}} 事件之前触发的。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/index.md:28]:
```suggestion
- {{domxref("InputEvent.data")}} {{ReadOnlyInline}}
  - : 返回包含插入字符的字符串，如果更改没有插入文本（例如删除字符时）则该值为空字符串。
- {{domxref("InputEvent.dataTransfer")}} {{ReadOnlyInline}}
  - : 返回一个 {{domxref("DataTransfer")}} 对象，该对象包含有关要添加到可编辑内容，或从可编辑内容中删除的富文本或纯文本数据的信息。
- {{domxref("InputEvent.inputType")}} {{ReadOnlyInline}}
  - : 返回对可编辑内容所做更改的类型，例如插入、删除或格式化文本。
- {{domxref("InputEvent.isComposing")}} {{ReadOnlyInline}}
  - : 返回一个 {{JSxRef("Boolean")}} 值，表明该事件是否在触发 {{domxref("Element/compositionstart_event", "compositionstart")}} 事件之后且触发 {{domxref("Element/compositionend_event", "compositionend")}} 事件之前触发的。
```

File: files/zh-cn/web/api/inputevent/index.md [hunk 4]
```diff
@@ -3,29 +3,36 @@ title: InputEvent
...
-- {{domxref("InputEvent.isComposing")}}{{readOnlyInline}}
-  - : 返回一个布尔值，表明该事件是在触发 [`compositionstart`](/zh-CN/docs/Web/API/Element/compositionstart_event) 事件之后且触发 [`compositionend`](/zh-CN/docs/Web/API/Element/compositionend_event) 事件之前触发的，也就是表明当前输入的字符是输入法的中途输入。
+## 实例方法
 
-## 方法
+_接口从其父级 {{domxref("UIEvent")}} 和 {{domxref("Event")}} 继承方法。_
 
-除继承自 {{domxref("UIEvent")}} 和 {{domxref("Event")}} 接口的方法外，没有其他自身方法。
+- {{DOMxRef('InputEvent.getTargetRanges()')}}
+  - : 返回一个 {{domxref("StaticRange")}} 数组，如果不取消输入事件，该数组将受到 DOM 更改的影响。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/index.md:35]:
```suggestion
  - : 返回一个 {{domxref("StaticRange")}} 对象数组，如果不取消输入事件，该数组将受到 DOM 更改的影响。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/inputevent/index.md (hunk 1 above)
Comment (fuchunhui) [reply files/zh-cn/web/api/inputevent/index.md:3]: thanks😀

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you! @fuchunhui 👍🏼

**Pull request merged**